app/models/eds_document.rb

```diff
@@ -351,7 +355,15 @@ def html_decode_and_sanitize(data, config = nil)
       html = html.gsub("\\n ", '')
     end
 
-    sanitized_html = Sanitize.fragment(html, sanitize_config)
+    begin
+      sanitized_html = Sanitize.fragment(html, sanitize_config)
+    rescue ArgumentError => e
+      # We still want to notify honeybadger but can provide more information
+      # about the html
+      Honeybadger.notify(e, error_message: "Error with arguments passed to Sanitize/Nokogiri", context: { html: })
+      # Replacing the html with an empty string to allow the page to still load
+      sanitized_html = ""
```

Comment:
We can just return the fallback, right? There's no need to do any of the other processing with the empty string or anything.

```suggestion
     return ''
```